packages/simcore-sdk/src/simcore_sdk/node_ports_v2/ports_mapping.py

```diff
@@ -1,15 +1,15 @@
-from typing import Dict, ItemsView, Iterator, KeysView, Union, ValuesView
+from typing import Dict, ItemsView, Iterator, KeysView, Type, Union, ValuesView
 
 from models_library.services import PROPERTY_KEY_RE
 from pydantic import BaseModel, constr
 
 from ..node_ports.exceptions import UnboundPortError
 from .port import Port
 
-PortKey: constr = constr(regex=PROPERTY_KEY_RE)
+PortKey: Type[str] = constr(regex=PROPERTY_KEY_RE)
```

Comment:
I do not understand this change here

Reply:
- you use types for annotations
- ``constr`` is a function-factory that creates constraint string **types** but not a type itself
If still unclear, look at pydantic code